src/main/java/com/teragrep/pth_06/planner/walker/Util.java

```diff
@@ -213,8 +218,14 @@ public static String elementAsString(Element el) {
             str = buffer.toString();
         }
         catch (TransformerConfigurationException tex) {
+            if (LOGGER.isDebugEnabled()) {
```

Comment:
catching exceptions and doing nothing with them is not ideal; I would make a refactoring issue about this class.

Reply:
Created new issue #112 about this.